src/ide.rs

```diff
@@ -74,6 +74,7 @@ pub fn parse_string(input: &[u8]) -> Result<String, serde_json::Error> {
     serde_json::from_str(&s)
 }
 
+// TODO: deprecate/remove in favor of ls_types::SymbolKind ?
```

Comment:
OK, could you add to the TODO that it is only used by the http server please? (We plan to remove the http mode soon-ish).